src/ol/webgl/Helper.js

```diff
@@ -805,22 +798,20 @@ class WebGLHelper extends Disposable {
   }
 
   /**
-   * Use a program.  If the program is already in use, this will return `false`.
+   * Set up a program for use. The program will be set as the current one. Then, the uniforms used
+   * in the program will be set based on the current frame state and the helper configuration.
    * @param {WebGLProgram} program Program.
-   * @return {boolean} Changed.
+   * @param {import("../PluggableMap.js").FrameState} frameState Frame state.
    * @api
```

Comment:
Here is an example of where it was premature to add this to the `@api`.  This is technically a breaking change (adding a required argument).

I think that the helper and WebGL layer renderers should be internal for now.  This gives us the ability to continue improving the higher level API (layers and sources) without breaking applications.